### PR TITLE
chore: refactor configs for easy volume mounting

### PR DESCRIPTION
**Pull request by nkoppisetty**
Explain what this PR does.
1. Moved _config.py out of config directory for easy ConfigMap mounting
2. Renamed default configs and metric names according to standardized names
3. Added app and rollouts_pod_template_hash as labels for default rollout metrics
4. Fixed postprocess - removed hardcoded labels
5. Fixed trainer to work with rollouts_pod_template_hash label
6. Fixed and added more test cases

### CODE REVIEW

File: numaprom/__init__.py [hunk 1]
```diff
@@ -21,3 +23,6 @@ def get_logger(name):
     pl_logger.setLevel(logging.ERROR)
     pl_logger.addHandler(stream_handler)
     return logger
+
+
+__all__ = ["UnifiedConf", "MetricConf", "ServiceConf", "NumapromConf", "get_logger"]
```
Comment (ab93) [inline numaprom/__init__.py:28]: Are we importing this `get_logger` anywhere?

File: numaprom/udsink/train_rollout.py [hunk 2]
```diff
@@ -123,14 +123,20 @@ def train_rollout(datums: List[Datum]) -> Responses:
 
         model_cfg = metric_config.numalogic_conf.model
 
+        # ToDo: standardize the label name
+        if "rollouts_pod_template_hash" in payload.composite_keys.keys():
```
Comment (ab93) [inline numaprom/udsink/train_rollout.py:127]:
```suggestion
        if "rollouts_pod_template_hash" in payload.composite_keys:
```

File: tests/resources/config.yaml [hunk 3]
```diff
@@ -26,44 +26,13 @@ configs:
...
     metric_configs:
-      - metric: "rollout_error_rate"
-        composite_keys: [ "namespace", "name", "hash_id" ]
+      - metric: "namespace_app_rollouts_http_request_error_rate"
+        composite_keys: [ "namespace", "name", "app", "rollouts_pod_template_hash" ]
+        static_threshold: 3
+      - metric: "namespace_app_rollouts_http_request_latency"
+        composite_keys: [ "namespace", "name", "app", "rollouts_pod_template_hash" ]
         static_threshold: 3
-        scrape_interval: 30
```
Comment (ab93) [inline tests/resources/config.yaml:32]: Any reason why we are changing the config file in tests? Some tests used to run on the basis of the config here

**Review by ab93 — COMMENTED** [review]
Some comments

File: tests/resources/config.yaml (hunk 3 above)
Comment (nkoppisetty) [reply tests/resources/config.yaml:32]: yes, this is to test the default rollout the config. If you are adding a new config, better to add it to test config and test it out against stream.json

File: numaprom/__init__.py (hunk 1 above)
Comment (nkoppisetty) [reply numaprom/__init__.py:28]: yes, everywhere

**Review by ab93 — APPROVED** [review]

File: numaprom/__init__.py (hunk 1 above)
Comment (vigith) [reply numaprom/__init__.py:28]: won't this `logging.getLogger(__name__)` do?

**Review by vigith — APPROVED** [review]

File: numaprom/__init__.py (hunk 1 above)
Comment (nkoppisetty) [reply numaprom/__init__.py:28]: for setting pytorch_lightning logs we had to make it a method call it explicitly

**Pull request merged**